src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.PowerFx.Preview
     public static class FeatureFlags
     {
         private static bool _stringInterpolation = false;
+        private static bool _disableAtTableField = false;
```

Comment:
>_disableAtTableField [](http://example.com/codeflow?start=28&length=20)

This should actually be on 

[https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs](https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs)

Those are per-instance

These are static and will be deprecated